ext/standard/head.c

```diff
@@ -120,7 +120,7 @@ PHPAPI zend_result php_setcookie(zend_string *name, zend_string *value, time_t e
 
 	/* Should check value of SameSite? */
 
-	if (value == NULL || ZSTR_LEN(value) == 0) {
+	if (value == NULL || atol(ZSTR_VAL(value)) < 0 || ZSTR_LEN(value) == 0) {
```

Comment:
Shouldn't this be `expires < 0`? Though warning would be preferred anyway.

Reply:
We will accept with notice, thanks for reviewing.